test_suite/tests/test_macros.rs

```diff
@@ -1086,6 +1086,34 @@ fn test_adjacently_tagged_enum() {
         ],
     );
 
+    // unit with no content (integer tag)
+    assert_de_tokens(
+        &AdjacentlyTagged::Unit::<u8>,
+        &[
+            Token::Struct {
+                name: "AdjacentlyTagged",
+                len: 2,
+            },
+            Token::U64(0),
```

Comment:
I would add comments to tokens that represents tag and content, for clarify
```suggestion
            Token::U64(0), // tag
```